src/lib/source-handlers/azure/list-projects.ts

```diff
@@ -0,0 +1,116 @@
+import { AzureProjectData as Project } from './types';
+import base64 = require('base-64');
+import Bottleneck from 'bottleneck';
+import * as debugLib from 'debug';
+import { AzureProjectData } from './types';
+import { getAzureToken } from './get-azure-token';
+import * as needle from 'needle';
+
+const debug = debugLib('snyk:azure');
+const azureDefaultUrl = 'https://dev.azure.com';
+
+const limiter = new Bottleneck({
+  maxConcurrent: 1,
+  minTime: 500,
+});
+
+limiter.on('failed', async (error, jobInfo) => {
+  const id = jobInfo.options.id;
+  console.warn(`Job ${id} failed: ${error}`);
```

Comment:
this should be `debug` as per previous comment

Reply:
Done